data/boarding missions.txt

```diff
@@ -37,3 +37,29 @@ mission "Assisting Hai"
 			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
 				decline
 	destination Hai-home
+
+mission "Navy emergency"
+	assisting
+	repeat
+	deadline
+	passengers 1 3
+	description "Deliver <fare> to <destination> by <date>; the military hospital is waiting for you."
+	to offer
+		"reputation: Navy (Oathkeeper)" > 100
+		random < 15
+	source
+		government "Navy (Oathkeeper)"
+		system "Arneb" "Hatysa" "Alnilam" "Unagi" "Almaaz" "Mintaka" "Gorvi" "Tortor" "Fah Root" "Rigel" "Hassaleh" "Danoa" "Rajak" "Sumar" "Ultima Thule" "Cardax" "Volax" "Moktar"
+	destination "Farpoint"
+	on offer
+		conversation
+			`When you repair the <origin>, the captain thanks you, but adds, "We had injuries during the battle. Could you bring the injured to the military hospital on <destination>? It would only be <bunks> <passengers>."`
```

Comment:
`<bunks> <passengers>` should be `<fare>`.
Also, why can't they bring the injured there themselves? You did just repair them.